[2023-09-12 04:34:48,839] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: CallPostgreOperator.get_data scheduled__2023-09-12T00:00:00+00:00 [queued]>
[2023-09-12 04:34:48,850] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: CallPostgreOperator.get_data scheduled__2023-09-12T00:00:00+00:00 [queued]>
[2023-09-12 04:34:48,850] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-12 04:34:48,850] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-09-12 04:34:48,850] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-12 04:34:48,865] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_data> on 2023-09-12 00:00:00+00:00
[2023-09-12 04:34:48,869] {standard_task_runner.py:52} INFO - Started process 185 to run task
[2023-09-12 04:34:48,871] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'CallPostgreOperator', 'get_data', 'scheduled__2023-09-12T00:00:00+00:00', '--job-id', '947', '--raw', '--subdir', 'DAGS_FOLDER/CallPostgreOperator.py', '--cfg-path', '/tmp/tmp_28v3w2_', '--error-file', '/tmp/tmpm30x5k_4']
[2023-09-12 04:34:48,871] {standard_task_runner.py:80} INFO - Job 947: Subtask get_data
[2023-09-12 04:34:48,934] {task_command.py:371} INFO - Running <TaskInstance: CallPostgreOperator.get_data scheduled__2023-09-12T00:00:00+00:00 [running]> on host b18943745c0d
[2023-09-12 04:34:48,967] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-09-12 04:34:49,004] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=CallPostgreOperator
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2023-09-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-12T00:00:00+00:00
[2023-09-12 04:39:11,248] {python.py:173} INFO - Done. Returned value was: None
[2023-09-12 04:39:11,265] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=CallPostgreOperator, task_id=get_data, execution_date=20230912T000000, start_date=20230912T043448, end_date=20230912T043911
[2023-09-12 04:39:11,312] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-09-12 04:39:11,349] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2023-09-12 05:07:39,706] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: CallPostgreOperator.get_data scheduled__2023-09-12T00:00:00+00:00 [queued]>
[2023-09-12 05:07:39,715] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: CallPostgreOperator.get_data scheduled__2023-09-12T00:00:00+00:00 [queued]>
[2023-09-12 05:07:39,715] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2023-09-12 05:07:39,715] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2023-09-12 05:07:39,715] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2023-09-12 05:07:39,729] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): get_data> on 2023-09-12 00:00:00+00:00
[2023-09-12 05:07:39,733] {standard_task_runner.py:52} INFO - Started process 1955 to run task
[2023-09-12 05:07:39,735] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'CallPostgreOperator', 'get_data', 'scheduled__2023-09-12T00:00:00+00:00', '--job-id', '1052', '--raw', '--subdir', 'DAGS_FOLDER/CallPostgreOperator.py', '--cfg-path', '/tmp/tmp_zwbug2d', '--error-file', '/tmp/tmp1zy9g5zh']
[2023-09-12 05:07:39,736] {standard_task_runner.py:80} INFO - Job 1052: Subtask get_data
[2023-09-12 05:07:39,794] {task_command.py:371} INFO - Running <TaskInstance: CallPostgreOperator.get_data scheduled__2023-09-12T00:00:00+00:00 [running]> on host b18943745c0d
[2023-09-12 05:07:39,826] {warnings.py:110} WARNING - /home/***/.local/lib/python3.7/site-packages/***/utils/context.py:202: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(key, self._deprecation_replacements[key]))

[2023-09-12 05:07:39,864] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=CallPostgreOperator
AIRFLOW_CTX_TASK_ID=get_data
AIRFLOW_CTX_EXECUTION_DATE=2023-09-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2023-09-12T00:00:00+00:00
[2023-09-12 05:07:39,865] {logging_mixin.py:115} INFO - 현재 전달된 url은 https://dumps.wikimedia.org/other/pageviews/2022/2022-09/pageviews-20220912-000000.gz 이다.
[2023-09-12 05:07:50,093] {python.py:173} INFO - Done. Returned value was: None
[2023-09-12 05:07:50,128] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=CallPostgreOperator, task_id=get_data, execution_date=20230912T000000, start_date=20230912T050739, end_date=20230912T050750
[2023-09-12 05:07:50,155] {local_task_job.py:156} INFO - Task exited with return code 0
[2023-09-12 05:07:50,194] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
